src/tag/Expression.js

```diff
@@ -28,8 +28,10 @@ export function Expression(tagName, typeValue) {
                 return `Array.isArray(${tagName})`;
             }
             return `(
+                typeof Symbol === "function" && typeof Symbol.hasInstance === "symbol" && !!${expectedName} && typeof ${expectedName}[Symbol.hasInstance] === "function" ?
```

Comment:
` !!${expectedName}` should be ` typeof ${expectedName} !== "undefined"`

`expectedName` may throw ReferenceError.